Mathlib/MeasureTheory/Integral/SetIntegral.lean

```diff
@@ -1379,6 +1379,29 @@ variable {X Y F G 𝕜 : Type*} [TopologicalSpace X]
 
 open Metric ContinuousLinearMap
 
+/-- The parametric integral over a continuous function on a compact set is continuous,
+  under mild assumptions on the topologies involved. -/
+theorem continuous_parametric_integral_of_continuous
+    [FirstCountableTopology X] [LocallyCompactSpace X]
+    [OpensMeasurableSpace Y] [SecondCountableTopologyEither Y E] [IsLocallyFiniteMeasure μ]
+    {F : X → Y → E} (hF : Continuous fun p : X × Y ↦ F p.1 p.2) {s : Set Y} (hs : IsCompact s) :
+    Continuous fun x ↦ ∫ a in s, F x a ∂μ := by
+  rw [continuous_iff_continuousAt]
+  intro x₀
+  rcases exists_compact_mem_nhds x₀ with ⟨U, U_cpct, U_nhds⟩
+  rcases (U_cpct.prod hs).bddAbove_image hF.norm.continuousOn with ⟨M, hM⟩
+  apply continuousAt_of_dominated
+  · exact eventually_of_forall fun x ↦ (hF.comp (Continuous.Prod.mk x)).aestronglyMeasurable
+  · refine Eventually.mono U_nhds fun x x_in ↦ ?_
+    rw [ae_restrict_iff]
+    · exact eventually_of_forall fun t t_in ↦ hM (mem_image_of_mem _ <| mk_mem_prod x_in t_in)
+    · exact (isClosed_le (hF.comp <| Continuous.Prod.mk x).norm continuous_const).measurableSet
+  · exact integrableOn_const.mpr (Or.inr hs.measure_lt_top)
+  · apply ae_of_all
+    intro a
```

Comment:
Just wondering: can any of this be streamlined with the `filter_upwards` tactic?

Reply:
Thanks for asking! I managed to use it thrice (in the last commit). I'm not sure if all of these are better or if the "using" lines should rather be separate exact statements. Let me know if you prefer a different style!